examples/linchpin_openstack_create.tmpl

```diff
@@ -7,13 +7,15 @@ simple:
         resource_group_type: openstack
         resource_definitions:
           - name: {{ .name }}
+            count: 1
```

Comment:
How this file related to the PR?

Reply:
Needed to make it work to test the execution, it can not run without the count, I can open a separate pr for it if we don't want to keep it here, but I think it doesn't hurt as it's just a small fix